### PR TITLE
Allow namespace to be set by programs consuming helm.

### PR DESCRIPTION
**Pull request by aaronmell**
Signed-off-by: Aaron Mell <amell@lumindigital.com>
When consuming the helm client, there is no way to correctly set the namespace except via an env variable, and that doesn't work with terraform possibly running the client concurrently in multiple namespaces. 

I'm not sure if you want to expose Namespace. If thats the case, maybe a SetNamespace function?

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
Would you mind writing a unit test to cover this?

**Comment by aaronmell** [comment]
Sure, were you thinking a test to verify that the namespace is correct? or something else?

**Comment by thomastaylor312** [comment]
Labeled as breaking because it is changing a publicly exported function signature

**Comment by aaronmell** [comment]
@bacongobbler @thomastaylor312 I reworked my change to reduce the scope, I think its a better way to fix it than changing the Namespace scope.

File: cmd/helm/helm.go [hunk 1]
```diff
@@ -67,7 +67,12 @@ func main() {
 	actionConfig := new(action.Configuration)
 	cmd := newRootCmd(actionConfig, os.Stdout, os.Args[1:])
 
-	if err := actionConfig.Init(settings, false, os.Getenv("HELM_DRIVER"), debug); err != nil {
+	// var namespace string
```
Comment (thomastaylor312) [inline cmd/helm/helm.go]: I assume this commented code doesn't need to be here?

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
This does look much cleaner. Just one thing to remove

**Review by thomastaylor312 — APPROVED** [review]
I'll let @bacongobbler give this one more look because he started review

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**